testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/oauth/OIDCProtocolMappersTest.java

```diff
@@ -23,6 +23,7 @@
 import org.junit.Test;
 import org.keycloak.admin.client.resource.ClientResource;
 import org.keycloak.admin.client.resource.ClientScopeResource;
+import org.keycloak.admin.client.resource.GroupResource;
```

Comment:
@jurjanpaul please, can you remove this unused import, besides that tests are clearly described and pass without any issue.

Reply:
@lhanusov Removed. Thanks for noticing!